sycl/source/detail/scheduler/commands.cpp

```diff
@@ -128,6 +128,9 @@ Command::Command(CommandType Type, QueueImplPtr Queue, bool UseExclusiveQueue)
   MEvent.reset(new detail::event_impl());
   MEvent->setCommand(this);
   MEvent->setContextImpl(detail::getSyclObjImpl(MQueue->get_context()));
+  if (MQueue->is_host() &&
```

Comment:
Why are you doing it here? I think this code should be in event_impl c'tor.

Reply:
Moved this to event_impl c'tor.